scripts/manylinux/build_manylinux_wheel.sh

```diff
@@ -0,0 +1,183 @@
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+set -o errexit
+
+
+USAGE="Usage: `basename ${0}` [-b Release | Debug]
+
+This script runs inside numenta's custom manylinux docker container and builds
+the manylinux x86_64 wide-unicode nupic.bindings wheel per PEP-513. See
+https://github.com/numenta/manylinux.
+
+INPUT: Expects a pristine nupic.core source tree without any remnant build
+   artifacts from prior build attempts. Otherwise, behavior is undefined.
+
+OUTPUT: On success, the resulting manylinux wheel will be located in the
+  subdirectory nupic_bindings_wheelhouse of the source tree's root directory.
+
+OPTIONS:
+-h                  : print command help.
+-b Release | Debug  : Specifies build type, which may be either Release or Debug;
+                      defaults to Release. [OPTIONAL]
+-k build-result-key : Build result key; Used to decorate artifact names.
+                      [OPTIONAL]
+"
+
+
+# Parse command line arguments
+
+# Build type may be Release or Debug; defaults to Release
+BUILD_TYPE="Release"
+
+RESULT_KEY=""
+
+while getopts ":hb:k:" opt; do
+  case $opt in
+    h)
+      echo "${USAGE}"
+      exit 0
+      ;;
+    b)
+      if [[ "$OPTARG" == "Release" || "$OPTARG" == "Debug" ]]; then
+        BUILD_TYPE=$OPTARG
+      else
+        echo "-$opt was triggered with invalid parameter: $OPTARG" >&2
+        exit 1
+      fi
+      ;;
+    k)
+      RESULT_KEY="$OPTARG"
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      exit 1
+      ;;
+    :)
+      echo "Option -$OPTARG requires an argument." >&2
+      exit 1
+      ;;
+  esac
+done
+
+
+set -o xtrace
+
+
+NUPIC_CORE_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
+
+DEST_WHEELHOUSE="${NUPIC_CORE_ROOT}/nupic_bindings_wheelhouse"
+
+WHEEL_PLATFORM_NAME="manylinux1_x86_64"
+
+
+echo "RUNNING MANYLINUX BUILD: BUILD_TYPE=${BUILD_TYPE}, ROOT=${NUPIC_CORE_ROOT}"
+
+# The manylinux image provides cmake28, but doesn't symlink it to cmake
+CMAKE="cmake28"
+${CMAKE} --version
+
+# manylinux provides the required gcc toolchain for building extensions
+CC="gcc"
+CXX="g++"
+
+# Add the python 2.7 binaries from manylinux image to PATH, overriding system
+# Python
+PYBIN="/opt/python/cp27-cp27mu/bin"
+
+PATH="${PYBIN}:${PATH}"
+
+# Help python executable find its shared lib; unlike the community manylinux
+# docker image that builds statically-linked python, our custom manylinux image
+# builds python with libpython.so python library, because our current C++ tests
+# of python interfaces depend on having libpython.so.
+LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:$(dirname ${PYBIN})/lib"
+
+# Install pycapnp to get the headers capnp/helpers/checkCompiler.h, etc.
+${PYBIN}/pip install pycapnp==0.5.8
```

Comment:
> You added ${PYBIN} to the PATH so no need to prefix the executable with it, right?

I was following the manylinux example. But you're right, since I prepended it to PATH, the explicit PYBIN is redundant.